spec/models/user_spec.rb

```diff
@@ -23,6 +23,14 @@
     end
   end
 
+  context 'authored_notes' do
+    it 'returns all authored_notes for the user' do
+      expect(user.authored_notes).to be_empty
```

Comment:
A method I didn't need, but rubocop forced me to do it.

Reply:
Every time Rubocop suggests something you know you don't need, that's an opportunity to disable it. Although learning about the cop itself about why it exists and how it solves a particular problem is also worth taking the time.

Reply:
Alright, I am removing the user.authored_notes and disabling the cop for Note#author's `belongs_to` method.